doc/lsst.afw.cameraGeom/cameraGeom.rst

```diff
@@ -0,0 +1,147 @@
+
+.. _subsection_CameraGeom_Camera:
+
+Camera
+------
+
+Camera is a collection of :ref:`Detectors <subsection_CameraGeom_Detector>`.
+
+Camera also supports coordinate transformation between all :ref:`camera coordinate systems <section_Camera_Coordinate_Systems>`.
+
+.. _subsection_CameraGeom_Detector:
+
+Detector
+--------
+
+Detector contains information about a given imaging detector (typically a CCD), including its position and orientation in the focal plane and information about amplifiers (such as the image region, overscan and readout corner).
+Amplifier data is stored as records in an :py:class:`lsst.afw.table.AmpInfoTable`, and Detector acts as a collection of :py:class:`lsst.afw.table.AmpInfoRecord`.
+
+Detector also supports transformation between :ref:`FOCAL_PLANE <CameraGeom_FOCAL_PLANE>`, :ref:`PIXELS <CameraGeom_PIXELS>`, and (if a suitable transform has been provided) :ref:`ACTUAL_PIXELS <CameraGeom_ACTUAL_PIXELS>` coordinates.
+However Detector does *not* support :ref:`FIELD_ANGLE <CameraGeom_FIELD_ANGLE>` coordinates; use a :py:class:`lsst.afw.cameraGeom.Camera` for that.
+
+.. _subsection_CameraGeom_CameraSys_and_CameraSysPrefix:
+
+CameraSys and CameraSysPrefix
+-----------------------------
+
+CameraSys represents a :ref:`camera coordinate system <section_Camera_Coordinate_Systems>`. It contains
+a coordinate system name and a detector name. The detector name is blank for non-detector-based
+:ref:`camera coordinate systems <section_Camera_Coordinate_Systems>` such as
+:ref:`FOCAL_PLANE <CameraGeom_FOCAL_PLANE>` and :ref:`FIELD_ANGLE <CameraGeom_FIELD_ANGLE>`,
+but must always name a specific detector for detector-based coordinate systems.
+
+CameraSysPrefix is a specialized variant of CameraSys that represents a detector-based coordinate system
+when the detector is not specified. CameraSysPrefix contains a coordinate system name but no detector name.
+
+A constant is provided each :ref:`camera coordinate system <section_Camera_Coordinate_Systems>`:
+
+* __FOCAL_PLANE__ (a CoordSys) for the :ref:`FOCAL_PLANE <CameraGeom_FOCAL_PLANE>` system
+* __FIELD_ANGLE__ (a CoordSys) for the :ref:`FIELD_ANGLE <CameraGeom_FIELD_ANGLE>` system
+* __PIXELS__ (a CoordSysPrefix) for the :ref:`PIXELS <CameraGeom_PIXELS>` system
+* __ACTUAL_PIXELS__ (a CoordSysPrefix) for the :ref:`ACTUAL_PIXELS <CameraGeom_ACTUAL_PIXELS>` system
+
+All Detector methods that take a \ref cameraGeom::CameraSys "CameraSys" also accept a \ref cameraGeom::CameraSysPrefix "CameraSysPrefix" instead.
+For example to transform a list of points from :ref:`PIXELS <CameraGeom_PIXELS>` to :ref:`FOCAL_PLANE <CameraGeom_FOCAL_PLANE>` system using a :py:class:`lsst.afw.cameraGeom.Detector`:
+
+.. code-block:: python
+
+    focalPlanePoints = Detector.transform(pixelPoints, PIXELS, FOCAL_PLANE)
+
+Camera methods always require a ``CameraSys``; a ``CameraSysPrefix`` is not acceptable because the camera does not know which detector to use.
```

Comment:
If my suggestion above works pleas change the double back-ticks to single back-ticks